myuw/templates/handlebars/card/registration/reg_resources.html

```diff
@@ -1,46 +1,61 @@
 {% load templatetag_handlebars %}
 {% tplhandlebars "reg_resources_tmpl" %}
 
-<div class="card-badge-container reg_resources_disc">
-
-    <h4 class="sr-only">Registration resources</h4>
-
-    <ul class="reg-resources-list no-header-space">
-
-        {{#if registration_is_open}}
-            <li>
-                {{#if plan_data}}
-                    {{#unless plan_data.has_ready_courses}}
-                        <a target="_blank" class="focus-link"
-                           href="{{plan_data.myplan_href}}">Edit plan in MyPlan</a>
-                    {{else}}
-                        <a target="_blank" class="focus-link"
-                           href="{{plan_data.registration_href}}">Register using MyPlan (Non-PCE courses only)</a>
-                    {{/unless}}
+    {{#if registration_is_open}}
+    <div class="card-actions-wrapper">
+        <div class="btn btn-default card-action-btn card-action-btn-first">
+            {{#if plan_data}}
+                {{#unless plan_data.has_ready_courses}}
+                        <a target="_blank" class="focus-link" data-linklabel="Edit Plan in MyPlan"
+                           href="{{plan_data.myplan_href}}">Edit Plan in MyPlan</a>
                 {{else}}
-                    <a target="_blank" class="focus-link"
-                       href="https://uwstudent.washington.edu/student/myplan/mplogin/netid?rd=/student/myplan/registration/{{reg_next_year}}{{get_quarter_code reg_next_quarter}}">Register using MyPlan (Non-PCE courses only)</a>
-                {{/if}}
-            </li>
+                        <a target="_blank" class="focus-link" data-linklabel="Register Using MyPlan"
+                           href="{{plan_data.registration_href}}">Use MyPlan to Register</a>
+                {{/unless}}
+            {{else}}
+                <a target="_blank" class="focus-link"
+                   href="https://uwstudent.washington.edu/student/myplan/mplogin/netid?rd=/student/myplan/registration/{{reg_next_year}}{{get_quarter_code reg_next_quarter}}">Use MyPlan to Register</a>
+            {{/if}}
+        </div>
 
-            <li class="focus-link last-focus-link">
+        <div class="btn btn-default card-action-btn">
+            <a target="_blank"
+               class="reg-link"
+               href="https://sdb.admin.uw.edu/students/uwnetid/register.asp">Register using SLN codes</a>
+        </div>
+        {{#if is_c2}}
+            <br>
+            <span id="pce_registration"
+                  class="sr-only">
+                Learn how to register for PCE courses
```

Comment:
I don't really get why this sr-only content is even needed. It says the exact same thing as the link it is labelling...